src/promptflow/promptflow/_sdk/_utils.py

```diff
@@ -625,6 +626,24 @@ def _generate_tool_meta(
     return res
 
 
+def _get_dynamic_list(function_config: Dict) -> List:
+    func_path = function_config.get("func_path", type=str)
+    func_kwargs = function_config.get("func_kwargs", type=dict)
+    # parameter
+    # retrieve ws triple, system append to func_kwargs.
+    # Question: inappropriate to ref cli internal method in  sdk?
```

Comment:
@brynn-code could you please take a look at this question? much appreciated if better idea could be shared.

Reply:
Some concern about this:
1. _sdk shall not import _cli 's function
2. This will read from ~/.azure and requires user installed azure cli, what if user don't install it? How to config the workspace triple?